src/app/index.ts

```diff
@@ -258,7 +257,11 @@ const loginRedirect = (req: express.Request, res: express.Response, _: express.N
   const parsedUrl = parse(req.originalUrl);
   const searchParam = parsedUrl.search;
   if (searchParam) {
-    const searchString = trimStart(searchParam, '?');
+    let searchString = searchParam;
+    // remove the leading '?'
+    if (searchParam.charAt(0) === '?') {
+      searchString = searchParam.replace('?', '');
```

Comment:
@machariamuguku would being a bit more defensible and remove only '?' at the start be reasonable

Reply:
nevermind, 
actually seems like replace only replaces the first occurence